### ISSUE TITLE
Ubuntu 16.04 install stopping short

### ISSUE THREAD

**Issue by gacott** [nodemailer/wildduck#74]
Trying an install multiple times, keeps ending at

Creating one for you again.
Setting up torsocks (2.1.0-2) ...
Setting up nodejs (4.2.6~dfsg-1ubuntu4.1) ...
update-alternatives: using /usr/bin/nodejs to provide /usr/bin/js (js) in auto mode
Setting up tor-geoipdb (0.3.2.10-1~xenial+1) ...
Processing triggers for libc-bin (2.23-0ubuntu10) ...
Processing triggers for systemd (229-4ubuntu21.2) ...

How do I get past this? Thanks

**Comment by uzerus** [comment]
Did you `apt update` and `apt upgrade` before wildduck installation?
It looks like it's stopping while installing dependencies with `apt`.

If you have 1 instance installed, you cannot install 2nd instance on the same host (it tells about "wildduck user exist etc.). 

This is full-automated-install-script or manual install?

**Comment by gacott** [comment]
Yes, did a full update and upgrade before.
No install prior to this, clean install.
Fully automated script.

**Comment by uzerus** [comment]
Can you upload full log here? it should be in /var/log/... idk the name of this log... maybe can you add `--debug` to install script (i'm not sure it's possible)

**Comment by gacott** [comment]
Yeah, don't see a log file that it would be. I'll try again with the debug, see what we get.

**Comment by gacott** [comment]
Yeah, no debug, no log that I can see, same darn thing happening. I don't think it's dependencies that are causing the issue, it just thinks it's done. I'm not that great with this kind of stuff to be honest.

**Comment by gacott** [comment]
Okay, decided to forget the script and try manual, much closer I  think. The backend is running, not really sure if everything is there or not. Working on the front end now and having this issue: npm WARN grunt-wait@0.1.0 requires a peer of grunt@~0.4.1 but none is installed. You must install peer dependencies yourself.
